### PR TITLE
Update changes for pipelines-1.4 branch

### PR DESCRIPTION
**Pull request by savitaashture**

### CODE REVIEW

**Comment by pthangad** [comment]
@savitaashture  should we expect another pr to master branch?

**Comment by savitaashture** [comment]
> @savitaashture should we expect another pr to master branch?

@praveen4g0 PR to master to update vote api and ui links https://github.com/openshift/pipelines-tutorial/pull/136

**Comment by pthangad** [comment]
@savitaashture   we don't need now parameter `TLSVERIFY` set to false, in buildah cluster task

**Review by pthangad — CHANGES_REQUESTED** [review]
`01_pipeline/04_pipeline.yaml`  we should remove this param now,  as we no longer require setting `TLSVERIFY` flag as false

File: README.md [hunk 1]
```diff
@@ -536,7 +534,7 @@ spec:
 Run following command to apply Trigger.
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/03_trigger.yaml
+$ oc create -f https://github.com/openshift/pipelines-tutorial/blob/pipelines-1.4/03_triggers/03_trigger.yaml
```
Comment (pthangad) [inline README.md]: change to `https://raw.githubusercontent.com/openshift/pipelines-tutorial/pipelines-1.4/03_triggers/03_trigger.yaml`

**Review by pthangad — CHANGES_REQUESTED** [review]
please do update `https://github.com/openshift/pipelines-tutorial/blob/pipelines-1.4/03_triggers/03_trigger.yaml` to `https://raw.githubusercontent.com/openshift/pipelines-tutorial/pipelines-1.4/03_triggers/03_trigger.yaml`

File: README.md (hunk 1 above)
Comment (savitaashture) [reply README.md]: Updated

**Review by pthangad — APPROVED** [review]

**Pull request merged**